src/wasm.rs

```diff
@@ -986,6 +1024,10 @@ mod test {
         FailingModule<GovMsg, Empty, Empty>,
     >;
 
+    fn mock_wasm_keeper() -> WasmKeeper<Empty, Empty> {
+        WasmKeeper::new()
+    }
```

Comment:
The reason is to provide a single initialization point of the keeper for all tests. In tests currently the following constructs are used (the last example is not jest covered by this function - out of the scope of this change):
```
let mut keeper = WasmKeeper::<Empty, Empty>::new();
```
```
let mut keeper = WasmKeeper::new();
```
```
let mut keeper =
    WasmKeeper::<Empty, Empty>::new_with_custom_address_generator(TestAddressGenerator {
         addr_to_return: expected_addr.clone(),
    });
```
and only accidentally (thanks to Rust type resolver) all keepers are of type `WasmKeeper::<Empty, Empty>`.

To make it clear what type of keeper is used, I have added the additional initialization function with precise type specification.

Reply:
Okay, I see. You are just helping type elision. I am not a fan of it, as I like to depend on type elision, but it is also not the worst - especially if elision fails, it might help. And it is optional, which makes it ok. But I will still not use it when `WasmKeeper::new()` works perfectly fine - it is more explicit, I would use it only when type elision fails to avoid turbo fish.
+ I would really rename it to `wasm_keeper()`. I find the idiom "if the name is the same as the type, it is just a free constructor of a type" to be very clear and reduces boilerplate (especially since the function is just a type elision helper). I don't like no nitpick naming, but this one kind of hurts me.